README.md

```diff
@@ -1 +1,23 @@
-# balena-es-version
\ No newline at end of file
+# @balena/es-version
+
+This module allows specifying a target es version for other modules to require, currently supported target versions:
+* es3 (default)
+* es5
+* es2015
+* es2016
+* es2017
+* es2018
+* es2019
+* es2020
+* esnext
+
+To use it first require it before anything else and set the desired version:
+```js
+// Must be set before the first require of an @balena/es-version module
+require('@balena/es-version').set('es5')
```

Comment:
Make we could have a "strict" option as well, so that the library logs or throws when only `closestVersionOver` can be used.